dashboard/app/controllers/aichat_controller.rb

```diff
@@ -40,7 +40,7 @@ def request_chat_completion(payload)
 
   private def has_required_params?
     begin
-      params.require([:newMessage, :storedMessages, :aichatParameters, :chatContext])
+      params.require([:newMessage, :aichatParameters, :chatContext])
```

Comment:
I think Alice has this change in a PR as well -- I think Rails treats a blank array as a missing param, which is now possible without the initial message.